wezterm-gui/src/termwindow/render.rs

```diff
@@ -724,11 +729,32 @@ impl super::TermWindow {
                 let mut bg = bg_color;
                 let mut bg_default = bg_is_default;
 
-                if attrs.reverse() {
+                // Check the line reverse_video flag and flip.
+                if (attrs.reverse() && !params.line.is_reverse())
+                    || (!attrs.reverse() && params.line.is_reverse())
+                {
                     std::mem::swap(&mut fg, &mut bg);
                     bg_default = false;
                 }
 
+                // Check for blink, and if this is the "not-visible"
+                // part of blinking then set fg = bg.  This is a cheap
+                // means of getting it done without impacting other
+                // features.
+                let blink_rate = match attrs.blink() {
+                    Blink::None => 0,
+                    Blink::Slow => params.config.text_blink_rate,
+                    Blink::Rapid => params.config.text_blink_rate_rapid,
+                };
+                if blink_rate != 0
+                    && (self.config.text_blink_rate != 0 || self.config.text_blink_rate_rapid != 0)
```

Comment:
`params.config` is the same as `self.config`, so these `!= 0` should be redundant; `blink_rate` will be set to the configured value and if it is non-zero it is because the config was non-zero. This conditional should be ok to be;

```rust
if blink_rate != 0 {
```